starboard/xb1/tools/xb1_launcher.py

```diff
@@ -88,8 +88,16 @@
 _ARGS_DIRECTORY = 'content/data/arguments'
 _STARBOARD_ARGUMENTS_FILE = 'starboard_arguments.txt'
 _DEFAULT_PACKAGE_NAME = 'GoogleInc.YouTube'
+_STUB_PACKAGE_NAME = 'Microsoft.Title.StubApp'
+_DEBUG_VC_LIBS_PACKAGE_NAME = 'Microsoft.VCLibs.140.00.Debug'
 _DEFAULT_APPX_NAME = 'cobalt.appx'
 _DEFAULT_STAGING_APP_NAME = 'appx'
+_EXTENSION_SDK_DIR = os.path.normpath(
```

Comment:
potentially better to use `realpath` here since we are invoking on windows:

https://docs.python.org/3/library/os.path.html#os.path.realpath

> Note This function emulates the operating system’s procedure for making a path canonical, which differs slightly between Windows and UNIX with respect to how links and subsequent path components interact.

IME, I have seen normpath fail on simple stuff like capitalizations. @oxve is my witness 😂

Reply:
Done. There's probably some other places we should swap that, but I'll handle that in a separate CL.